data/journal_check/bug_refs.json

```diff
@@ -20,10 +20,17 @@
         "description": "Directory '.*.hdmi' with parent 'vc4-hdmi.*' already present",
         "products": {
             "sle-micro": [ "5.0", "5.1" ],
-            "opensuse": ["15.3"]
+            "opensuse": ["Tumbleweed", "15.3", "15.2"]
         },
         "type": "bug"
     },
+    "bsc#1188270": {
+        "description": "lan78xx 1-1.1.1:1.0 eth0: No phy led trigger registered for speed\\(-1\\)",
+        "products": {
+            "opensuse": ["Tumbleweed"]
```

Comment:
we need to add 15.3 here as well https://openqa.opensuse.org/tests/1840709#step/journal_check/12